README.adoc

```diff
@@ -70,7 +70,7 @@ To install g2p-seq2seq :
 
 In case you have some trouble with dependencies, I have mirrored the last known working versions of them in the repository at : _(https://github.com/saurabhshri/mirror/)_ .
 
-The alternate ways of generating language models, dictionaries and grammars are covered later in the docs.
+The alternate ways of generating language models, dictionary and grammar are covered later in the docs.
```

Comment:
This is not ok. We are explicitely speaking about multiple instances of both the dictionary and grammar objects.